packages/vscode-messenger-devtools/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Change Log of `vscode-messenger-devtools`
 
 ## v0.4.5 (March 2023)
```

Comment:
Proposal: add information about the new cancellation support in v0.5.0 (Dec. 2024)

Reply:
It is not really related to the Devtools extension itself, but if we publish a new version of it I can add a note.

Reply:
Ah right, I overlooked that this is the devtool's changelog.